src/ChildProcess.php

```diff
@@ -50,25 +50,39 @@ public function all(): array
 
         return $hydrated;
     }
-
+        
     public function start(
+        string|array $cmd
         string $alias,
-        array $cmd,
         ?string $cwd = null,
         ?array $env = null,
         bool $persistent = false
     ): static {
+        $cwd = $cwd ?? base_path();
+
+        if (is_string($cmd)) {
+            // When a string is passed, explode it on the space
+            $cmd = array_values(array_filter(explode(' ', $cmd)));
+        }
+
         $process = $this->client->post('child-process/start', [
             'alias' => $alias,
             'cmd' => $cmd,
-            'cwd' => base_path(),
+            'cwd' => $cwd,
             'env' => $env,
             'persistent' => $persistent,
         ])->json();
 
         return $this->fromRuntimeProcess($process);
     }
 
+    public function artisan(string|array $cmd, string $alias, ?array $env = null, ?bool $persistent = false): self
```

Comment:
I really like this... maybe we can have a `php()` method too that doesn't force the `artisan` argument in there?